### ISSUE TITLE
Improve installation experience

### ISSUE THREAD

**Issue by miohtama** [collective/collective.developermanual#152]
We are still far from great

http://plone.293351.n2.nabble.com/First-time-try-to-install-plone-and-I-have-to-say-td7561373.html#a7561376

**Comment by muellert** [comment]
The link is invalid...

**Comment by svx** [comment]
here the mail copy and paste from google cache [sorry for the long post]:

<snip>
Hello, 

Coming from drupal , want to try plone. 

Centos 6.2 . 

Half an hour I'm trying to fight just to install the system , there is no tutorial for it. 
The problem is yum packages. 

[root@map22 Plone-4.2.1-UnifiedInstaller]# ./install.sh standalone --without-ssl 
Stand-Alone Zope Instance selected 

Detailed installation log being written to /root/Plone-4.2.1-UnifiedInstaller/install.log 

Unable to find libssl or openssl/ssl.h. 
libssl and its development headers are required for Plone. 
If you're sure you have these installed, and are still getting 
this warning, you may disable the libssl check by adding the 
--without-ssl flag to the install command line. 
Otherwise, install your platform's openssl-dev libraries and headers 
and try again. 

Why in 2012 Do I have to fight to install a cms? 

Why is it 50 megs where drupal is only 3 megs? 

I can't understand what is stand alone , Is it with no database?  with db , which one? 
It is hard to get into the system, the docs are not good . 

Now I'm stuck , can't even install it. 
</snip>

**Comment by miohtama** [comment]
My suggestion how to fix this

1) Add a new post-download page in plone.org 

2) When user clicks the installer link

AND

3) has Javasacript installer

4) We will redirect the user the post-install page

5) Which will trigger the download via <meta refresh> (think Sourceforge)

6) ... and the page has following information 

Plone comes with Python and ZODB. Download is big because of included dependencies.

Because of above you don't need Apache or MySQL or anything.

For the further installation instructions please read these links.

**Comment by cleder** [comment]
It might be an idea to create deb and rpm packages Plone-d
ependencies which do install the required packages but not Plone itself . As this would be a Meta package only they  could be included in the unified installer

**Comment by svx** [comment]
yeah :) but this will cover only rpm or deb based systems, and someone
will have to keep them up to date :)

another idea might be:

since the  Unified Installer is already doing checks
[https://github.com/plone/Installers-UnifiedInstaller/blob/master/preflight]
maybe add possible missing checks there and try to make the error
messages a bit more clear ?
i think this is more something for the installer right ? @smcmahon

  /
  /

**Comment by svx** [comment]
This will be hopefully solved with the new installer for Plone 5

**Comment by svx** [comment]
@sneridagh would something like that possible on the new plone.org

**Comment by sneridagh** [comment]
Certainly, I will add it to the feature/content list for plone.org.
Cheers.
V.

Víctor Fernández de Alba
http://about.me/victorfernandezdealba
g+/Twitter/IRC: sneridagh

**Comment by svx** [comment]
Great !
